$OpenBSD: patch-browser_app_profile_firefox_js,v 1.30 2019/12/16 09:17:22 landry Exp $

remove pledge configs from prefs, moved to external config files
https://bugzilla.mozilla.org/show_bug.cgi?id=1584839
https://hg.mozilla.org/mozilla-central/rev/396a73e240dfcb4273ec0657df07284b172d9ca6

Index: browser/app/profile/firefox.js
--- browser/app/profile/firefox.js.orig
+++ browser/app/profile/firefox.js
@@ -1137,11 +1137,7 @@ pref("dom.ipc.shims.enabledWarnings", false);
 #endif
 
 #if defined(XP_OPENBSD) && defined(MOZ_SANDBOX)
-  // default pledge strings for the main & content processes, cf bug 1457092
-  // broad list for now, has to be refined over time
-  pref("security.sandbox.pledge.main", "stdio rpath wpath cpath inet proc exec prot_exec flock ps sendfd recvfd dns vminfo tty drm unix fattr getpw mcast");
   pref("security.sandbox.content.level", 1);
-  pref("security.sandbox.pledge.content", "stdio rpath wpath cpath inet recvfd sendfd prot_exec unix drm ps");
 #endif
 
 #if defined(MOZ_SANDBOX)
